packages/shopping/src/controllers/user-order.controller.ts

```diff
@@ -66,6 +73,8 @@ export class UserOrderController {
       },
     },
   })
+  @authenticate('jwt')
+  @authorize({resource: 'order', scopes: ['find']})
```

Comment:
can we also add `voters: [compareId]` for patch and find methods?

Reply:
agreed